apps/interpreter/src/examples-smoke-test.spec.ts

```diff
@@ -74,7 +73,6 @@ describe('jv example smoke tests', () => {
     sqliteLoaderMock.restore();
 
     // Clear registries
-    clearBlockExecutorRegistry();
     clearConstraintExecutorRegistry();
```

Comment:
should we open an issue to remove `clearBlockExecutorRegistry()`?

Reply:
Why not just remove it?

Reply:
Ah I meant `clearConstraintExecutorRegistry`. I think we should do that but in another PR.

Reply:
Yeah, I think we should refactor that in a separate PR :+1:

Reply:
Decided to do it right away in this PR.